plugins/module_utils/network/sonic/utils/formatted_diff_utils.py

```diff
@@ -77,6 +86,45 @@ def get_key_match_op(key, test_keys):
     return k_match_op
 
 
+#
+# Pre-defined Merge Operations
+#
+
+
+"""
+Default key match operation: simply merge command to existing config.
+"""
+
+
+def __MERGE_OP_DEFAULT(key_set, command, exist_conf):
+    new_conf = exist_conf
+    trival_cmd_key_set, dict_list_cmd_key_set = get_key_sets(command)
+    nu, dict_list_exist_key_set = get_key_sets(new_conf)
+
+    for key in trival_cmd_key_set:
+        new_conf[key] = command[key]
+
+    common_dict_list_key_set = dict_list_cmd_key_set.intersection(dict_list_exist_key_set)
+    only_cmd_dict_list_key_set = dict_list_cmd_key_set.difference(common_dict_list_key_set)
```

Comment:
Keys present only in 'command' could be obtained as below: 

```suggestion
    only_cmd_dict_list_key_set = dict_list_cmd_key_set.difference(dict_list_exist_key_set)
```

Reply:
Done this. Thank you very much!

Reply:
This change isn't visible yet in the version of this file on the PR branch. Please commit/push the change to resolve this issue.